### PR TITLE
Building type view

### PR DESCRIPTION
**Pull request by pfeningviola**
Please, check it! :-)

### CODE REVIEW

File: Tribes/src/app/building-type/building-type.component.ts [hunk 1]
```diff
@@ -1,8 +1,16 @@
 import { BuildingService } from './../building.service';
+import { ResourceService } from './../resource.service';
 import { Building } from './../building';
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { Location } from '@angular/common';
+import { MAX_UPGRADE_LEVELS } from '../constants';
```
Comment (balazsbarni) [inline Tribes/src/app/building-type/building-type.component.ts]: u can import them i think in one line, with commas

File: Tribes/src/app/building.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
 export class Building {
     id: number;
-    type: string;
+    buldingTypeENUM: string;
```
Comment (balazsbarni) [inline Tribes/src/app/building.ts]: It is not just `type`?

Comment (pfeningviola) [reply Tribes/src/app/building.ts]: The json, which we get from the backend, has the field 'buildingTypeENUM'.

File: Tribes/src/app/constants.ts [hunk 3]
```diff
@@ -1,2 +1,8 @@
+export const TOKEN = 'TOKEN';
 export const ROOT_URL = 'http://localhost:8080';
+export const MAX_UPGRADE_LEVELS = 3;
+export const PRODUCTION_RATE = 10;
 export const COST_NEW_BUILDING = 250;
+export const COST_BASE_UPGRADE = 100;
+export const TOWNHALL_FOOD_CAPACITY = 50;
+export const TOWNHALL_GOLD_CAPACITY = 100;
```
Comment (symunona) [inline Tribes/src/app/constants.ts]: Very good!

File: Tribes/src/app/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -53,5 +53,4 @@ export class DashboardComponent implements OnInit {
     .subscribe(response => this.numberOfBarracks = response.buildingDTOS
       .filter(building => building.buldingTypeENUM === 'BARRACK').length);
   }
-
 }
```
Comment (symunona) [inline Tribes/src/app/dashboard/dashboard.component.ts]: When you do these formatting things, it increases the changes of conflicts of unrelated changes. 
The best practice usually is to have them in separate "code quality cleaning" tickets & PRs.
